charts/nextcloud/values.yaml

```diff
@@ -513,6 +526,26 @@ metrics:
 
   # podLabels: {}
 
+  nginx:
+    enabled: false
+    allow: []
+    # Example
+    # - 10.233.105.0/24
+    # - 10.43.0.0/16
+    service:
```

Comment:
This value has no affact anywhere ....
But maybe usefull (on deployment under ports and your ConfigMap rendered; or service).

Reply:
This is actually used in nginx-config.yaml:

```
            {{- range .Values.metrics.nginx.allow }}
                allow {{ . }};
            {{- end }}
```